rpcserver.go

```diff
@@ -4216,6 +4216,11 @@ func (r *rpcServer) extractPaymentIntent(rpcPayReq *rpcPaymentRequest) (rpcPayme
 		return payIntent, err
 	}
 
+	if len(rpcPayReq.PaymentAddr) != 32 {
```

Comment:
it's also valid to have a `PaymentAddr` of length 0

Reply:
Changed